source/content/guides/wordpress-configurations/04-wp-cfm.md

```diff
@@ -27,25 +27,25 @@ WP-CFM should only be used to write changes to code in Dev and Multidev environm
 
 ## Install and Deploy WP-CFM
 
-Each of the following steps can be done using the Pantheon and WordPress Dashboards or via the command line using Pantheon's CLI, [Terminus](/terminus):
+<TabList>
 
-1. [Set the connection mode to SFTP](/guides/sftp) for the Dev or Multidev environment via the Pantheon Dashboard or with Terminus:
+<Tab title=”Installing via Git Updater” id=”git-updater” active={true}>
 
-    ```bash{promptUser: user}
-    terminus connection:set <site>.<env> sftp
-    ```
+This method makes use of the [Git Updater](https://git-updater.com/) plugin to install and keep WP-CFM up-to-date. Follow the steps in [our documentation on installing plugins using Git Updater](/guides/wordpress-configurations/installing-updating-from-third-party-sources#using-git-updater-to-install-and-manage-plugins-and-themes-from-git-repositories) and use the following settings:
 
-1. Install the [WP-CFM](https://wordpress.org/plugins/wp-cfm/) plugin on the Dev Environment using the WordPress Dashboard or with Terminus:
+**Plugin URI:** `forumone/wp-cfm-dist`
+**Repository Branch:** `release`
+**Remote Repository Host:** GitHub
+**GitHub Access Token:** Blank 
 
-    ```bash{promptUser: user}
-    terminus wp <site>.<env> -- plugin install --activate wp-cfm
-    ```
+</Tab>
 
-1. Commit this change using the Site Dashboard or with Terminus:
+<Tab title=”Installing via Composer” id=”composer”>
+This method makes use of [Integrated Composer](/guides/integrated-composer) to manage installation and version management. This process assumes you already have a `composer.json` in your site repository and `build_step` is set to `true` in your `pantheon.yml`. Follow the steps in [our documentation on installing plugins using Composer](/guides/wordpress-configurations/installing-updating-from-third-party-sources#using-composer-to-source-plugins-and-packages) and use `forumone/wp-cfm-dist` as the repository.
```

Comment:
This should be `forumone/wp-cfm` when installing via Composer from Packagist.

Reply:
Yep, probably a find-replace error.